proxy.go

```diff
@@ -21,16 +21,21 @@ import (
 	"fmt"
 	"net"
 	"strconv"
+
+	"github.com/op/go-logging"
 )
 
 type Proxy struct {
 	conn net.Conn
 	Host string
 	Port uint32
+	log  *logging.Logger
 }
 
 func NewProxy(host string, port uint16) (*Proxy, error) {
-	p := &Proxy{}
+	p := &Proxy{
+		log: logging.MustGetLogger("exasol"),
```

Comment:
It appears that logging.MustGetLogger returns a new logger that is not the same logger as the one initialized in client.go so it defaults to outputting everything. So I guess we need to pass the c.log into NewProxy when it's called in bulk-api.go.

Reply:
> So I guess we need to pass the c.log into NewProxy when it's called in bulk-api.go.

Done. And added some documentation to `NewProxy` which handling this implies.